topics/community/faqs/topic_pages.md

```diff
@@ -5,18 +5,18 @@ layout: faq
 contributors: [nomadscientist, hexylena]
 ---
 
-You can find your **Community Page** and **Maintainer Page** by scrolling down on your training **Topic** page.
+You can find your **Community Home** and **Maintainer Home** by scrolling down on your training **Topic** page.
 
 > <hands-on-title>Go to your topic page resources</hands-on-title>
 >
 > 1. Go to any training topic of interest. This will have the form
 >    ```
 >     https://training.galaxyproject.org/training-material/topics/<your topic here>
 >    ```
-> 2. Add a #community  to the end of the url.
+> 2. Add a #community to the end of the url.
```

Comment:
```suggestion
> 2. Add a #community to the end of the URL (or scroll to the bottom of the topic page and hit the "Community Home" button)
```

(and optionally add a screenshot of the buttons?)

Reply:
I added an FAQ to tell them to do that, but I was trying to avoid screenshots for sustainability/accessibility reasons